person/person.py

```diff
@@ -0,0 +1,17 @@
+class Person(object):

```

Comment:
Why Not add `position` to `Person` Class and then assign it in the Classes that inherit the `Person` Class?